context-free.md

```diff
@@ -2,21 +2,32 @@
 
 ##  Who is this programming language for?
 
+Context Free Art is a programing language built for people that want to design compture generated art peices. This can be anything from graphic art to desgins. I would say that the primary focus for this language is for those who want a simple way to design art online. This being simpler then Python or Java.
```

Comment:
I like this point about Python or Java not being an easy way to make images compared to this DSL - they certainly can, but I've previously found it tricky to make graphics in those languages.